gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -663,10 +716,10 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
 
       fw = nnstreamer_filter_find (fw_name);
 
-      /* See if mandatory methods are filled in */
-      if (nnstreamer_filter_validate (fw)) {
```

Comment:
Note: This is removed as the framework is validated when the framework is probed.